routers/routes.js

```diff
@@ -5,6 +5,8 @@ const app = express.Router();
 
 app.get("/me", auth, controllers.loadUser);
 app.post("/login", controllers.login);
+app.get("/:id/profile", auth);
```

Comment:
Controller functionality isn't called?

Reply:
Controller is called in : app.put("/:id/profile", auth, controllers.updateProfile) and that happens when the user click's the update profile button. 

In our app.js we have a useEffect which loads the user when a dispatch ocurrs.

Reply:
So this route is not use in the app?

Reply:
Thanks Leonel, you are rigth the app was not using the route. I just commited the change.